language/bytecode_verifier/src/abstract_state.rs

```diff
@@ -185,6 +202,16 @@ impl AbstractState {
             }
         }
 
+        for (x, y) in &self.globals {
```

Comment:
nit, more informative local names than `(x, y)`

Reply:
There are many instances of this problem.  I will do a separate diff to fix all of them.